bundles/org.openhab.binding.modbus.e3dc/src/test/java/org/openhab/binding/modbus/e3dc/dto/DataBlockTest.java

```diff
@@ -62,21 +62,21 @@ public void testValidPowerBlock() {
         Optional<Data> dataOpt = mc.parse(DataType.POWER);
         assertTrue(dataOpt.isPresent());
         PowerBlock b = (PowerBlock) dataOpt.get();
-        assertEquals("242.0 W", b.pvPowerSupply.toString(), "PV Supply");
-        assertEquals("14.0 W", b.gridPowerSupply.toString(), "Grid Supply");
-        assertEquals("0.0 W", b.gridPowerConsumpition.toString(), "Grid Consumption");
-        assertEquals("303.0 W", b.batteryPowerSupply.toString(), "Battery Supply");
+        assertEquals("242 W", b.pvPowerSupply.toString(), "PV Supply");
+        assertEquals("14 W", b.gridPowerSupply.toString(), "Grid Supply");
+        assertEquals("0 W", b.gridPowerConsumpition.toString(), "Grid Consumption");
+        assertEquals("303 W", b.batteryPowerSupply.toString(), "Battery Supply");
     }
 
     @Test
     public void testValidPowerBlockNegativePVSupply() {
         Optional<Data> dataOpt = mcNegativePVSupply.parse(DataType.POWER);
         assertTrue(dataOpt.isPresent());
         PowerBlock b = (PowerBlock) dataOpt.get();
-        assertEquals("-330.0 W", b.pvPowerSupply.toString(), "PV Supply");
-        assertEquals("14.0 W", b.gridPowerSupply.toString(), "Grid Supply");
-        assertEquals("0.0 W", b.gridPowerConsumpition.toString(), "Grid Consumption");
-        assertEquals("303.0 W", b.batteryPowerSupply.toString(), "Battery Supply");
+        assertEquals("-330 W", b.pvPowerSupply.toString(), "PV Supply");
```

Comment:
I am not maintainer of modbus.e3dc but this seems like a harmless side effect of library change. I would not expect this to have negative side effects to non-test code.

Reply:
Yes `toString` values aren't the most future proof values to use in such comparisons.
They are easy to read and understand however. So that's why I opted for just updating them.